providers/netty4/src/main/java/org/asynchttpclient/netty/handler/StreamedResponsePublisher.java

```diff
@@ -0,0 +1,46 @@
+package org.asynchttpclient.netty.handler;
```

Comment:
Missing license header. Please grant copyright to "the AsyncHttpClient Project".